{
  "comments": [
    {
      "key": {
        "uuid": "94b6bd8e_d31f9b8a",
        "filename": "keyholder/daemon.py",
        "patchSetId": 4
      },
      "lineNbr": 67,
      "author": {
        "id": 2321
      },
      "writtenOn": "2018-11-13T01:59:32Z",
      "side": 1,
      "message": "Might be good to include this in the try statement otherwise you get something like:\n\n\u003e ValueError: not enough values to unpack (expected 3, got 1)",
      "revId": "0f91b1b5688ccf3118c56e900d305d3c36055b3e",
      "serverId": "e9e9afe9-4712-486d-8885-f54b72dd1951",
      "unresolved": false
    }
  ]
}